src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -47,12 +48,20 @@ class remove_virtual_functionst
     goto_programt::targett target);
 
   void get_functions(const exprt &, dispatch_table_entriest &);
-  void get_child_functions_rec(
+  void get_child_functions(
     const irep_idt &,
     const symbol_exprt &,
     const irep_idt &,
     dispatch_table_entriest &,
-    std::set<irep_idt> &visited) const;
+    std::set<irep_idt> &visited,
+    bool recurse=true) const;
+  void get_parent_functions(
```

Comment:
This reads like a duplicate of `resolve_concrete_function_callt` (see https://github.com/diffblue/cbmc/blob/develop/src/goto-programs/remove_virtual_functions.cpp#L300 for example)